### ISSUE TITLE
Unable to copy/select specific text from a response

### ISSUE THREAD

**Issue by shaunyogeshwaran** [h2oai/h2ogpt#1229]
Issue where you are unable to select a specific part of the response without having to resort to copying the whole response.

**Comment by llmwesee** [comment]
@shaunyogeshwaran thanks for raising this issue! I also wanted to raise the same issue.

**Comment by pseudotensor** [comment]
Yes it is annoying, I'll try to see what is going on.

**Comment by pseudotensor** [comment]
Unfortunately, even this most basic example in gradio has same issue:

```
import gradio as gr
import random
import time

with gr.Blocks() as demo:
    chatbot = gr.Chatbot()
    msg = gr.Textbox()
    clear = gr.ClearButton([msg, chatbot])

    def respond(message, chat_history):
        bot_message = random.choice(["How are you?", "I love you", "I'm very hungry"])
        chat_history.append((message, bot_message))
        time.sleep(2)
        return "", chat_history

    msg.submit(respond, [msg, chatbot], [msg, chatbot])

if __name__ == "__main__":
    demo.launch()


```

**Comment by pseudotensor** [comment]
It seems fixed in latest gradio==4.11.0 ,  I can highlight parts.

**Comment by pseudotensor** [comment]
solved now using gradio4
